packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -2190,7 +2192,8 @@ function commitDeletionEffectsOnFiber(
 
             let effect = firstEffect;
             do {
-              const {destroy, tag} = effect;
+              const tag = effect.tag;
+              const destroy = effect.inst.destroy;
```

Comment:
Btw why do we not clear out the destroy field here?

Reply:
I guess it doesn't really matter here because the fiber is being deleted but we can for consistency